packages/frontier-libraries/stdlib/package.json

```diff
@@ -38,7 +38,9 @@
     "type-fest": "^2.3.4"
   },
   "devDependencies": {
-    "@rdfrontier/lib-builder": "^1.0.0"
+    "babel-loader": "^8.2.5",
```

Comment:
Why is @rdfrontier/lib-builder removed and these other tools added. Cant imagine this project being buildable without this dependency